woodwork/tests/testing_utils/datatable_utils.py

```diff
@@ -6,18 +6,6 @@
 ks = import_or_none('databricks.koalas')
 
 
-def validate_subset_dt(subset_dt, dt):
```

Comment:
can we rename this file to `table_utils.py`?

Reply:
✅ 7c6b46e